Document/0x06b-Basic-Security-Testing.md

```diff
@@ -529,7 +529,7 @@ Writing bytes to destination...
 Successfully downloaded /var/mobile/Containers/Data/Application/72C7AAFB-1D75-4FBA-9D83-D8B4A2D44133/.com.apple.mobile_container_manager.metadata.plist to .com.apple.mobile_container_manager.metadata.plist
 ```
 
-You can also upload files with `file upload <local_file_path>` to the iOS device, but this implementation is not fully stable at the moment and might produce an error. If that's the case file an issue in the [objection GitHub repo](https://github.com/sensepost/objection/issues "Objection Issues").
+You can also upload files with `file upload <local_file_path>` to the iOS device, but this implementation is not fully stable at the moment and might produce an error. If that's the case, file an issue in the [objection GitHub repo](https://github.com/sensepost/objection/issues "Objection Issues").
```

Comment:
```suggestion
You can also upload files to the iOS device with `file upload <local_file_path>`, but this implementation is not fully stable at the moment and might produce an error. If that's the case, file an issue in the [objection GitHub repo](https://github.com/sensepost/objection/issues "Objection Issues").
```

Reply:
Isn't the unstable part always an issue? Don't we have a disclaimer somewhere at the start saying that tools can be broken and that they should open an issue / search github issues if that's the case? It also makes the guide very time specific, as it would have to be rewritten with every tool update and stuff like that.

I would remove the entire "but..." part.

Reply:
This is a very good point to address. This info was already there. And I'm afraid that we have this kind of sentences on more places in the guide.

I totally agree with your point, we should make it as timeless as possible. I'll check if we have a disclaimer, if not, we should definitely add one.

Reply:
I can't find it I'll add one proposal on a separate PR, it might need some discussion.